tpm/constants.go

```diff
@@ -92,6 +149,47 @@ const (
 	rtTrans
 )
 
+// Locality type
+type Locality byte
+
+// Values of locality
+// Note: Localities are summable
+const (
+	locZero Locality = 1 << iota
+	locOne
+	locTwo
+	locThree
+	locFour
+)
+
+// LocaMap maps Locality values to strings for convenience
+var LocaMap = map[Locality]string{
+	locZero:  "Locality 0",
+	locOne:   "Locality 1",
+	locTwo:   "Locality 2",
+	locThree: "Locality 3",
+	locFour:  "Locality 4",
+}
+
+// PrettyPrint for Localities
+func (l Locality) String() string {
+	var retString string
+	n, ok := LocaMap[l]
+	if !ok {
+		//In this case the given locality may a sum of localities
+		for iterator, item := range LocaMap {
+			l = l - iterator
+			if l == 0 {
+				retString += item
+				return retString
+			}
+			retString += item + " + "
+		}
+		return "unknown_locality"
```

Comment:
In the case of unknown bits, it might be useful to return the raw value, like this: 

```go
return fmt.Sprintf("locality<%d>", int(l))
```